src/fixed/layout.rs

```diff
@@ -14,26 +14,27 @@ pub(crate) enum LayoutModifiers {
     AltGr,
 }
 
-pub(crate) struct LayoutParser {
-    layout: HashMap<String, String>,
+/** This was renamed to `Layout` from `LayoutParser`*/
+pub(crate) struct Layout {
+    key_map: HashMap<String, String>,
```

Comment:
I think naming this `map` would suffice. No hard feelings though!

Reply:
No problem. Will do.